gems/quilt_rails/README.md

```diff
@@ -91,6 +93,9 @@ yarn add react react-dom
 yarn add typescript @types/react @types/react-dom
 ```
 
+🗒️ Ignore `sewing_kit` and `@shopify/sewing-kit` dependencies if you don't have access to it.
```

Comment:
Are these guides intended for internal and external devs?

Reply:
we will have to assume so no? In the future once gestalt is a thing we can point internal devs to that. For now this doc is for both since there is no place to redirect internal folks.